src/components/atoms/ProgressBar/ProgressBar.md

```diff
@@ -0,0 +1,3 @@
+## ProgressBar
+
+ProgressBar component is used to show the progress of the flow.
```

Comment:
Can you add a couple of examples of how to use it?

Reply:
it is added here https://bit.dev/campgladiator/cgui/components/atoms/progress-bar